docs/overrides/home.html

```diff
@@ -0,0 +1,151 @@
+{% extends "base.html" %}
+{% block tabs %}
+{{ super() }}
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="home.css">
+<link rel="stylesheet" href="home_responsive.css">
+<meta name="viewport" content="initial-scale=1, maximum-scale=1, width=device-width">
+
+<style>
+    
+</style>
+<div class="home">
+    <div class="Xournal" id="Xournal">
+		<div class="title">
+			<h1>Take handwritten notes with ease.</h1>
+			<h2>From solving a physics problem to making travel diaries.</h2>
+		</div>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful <span><a class ="Xournal-link" href="https://sourceforge.net/projects/xournal/">Xournal</a></span> program.</p>
+		<img src="img/Physics Problem.png">
+		<div class="download-hyperlink upper-button">
+			<a href="{{ 'installation/' | url }}">Download Xournal++</a>
+		</div>
+	</div>
+	
+	<div class="feature-list">
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Handwrite your notes</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Support for pressure-sensitive stylus and drawing tablets (Wacom, Huion, XP-Pen, etc.)</p>
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>Robust and customizable pen, highlighters, and eraser tool, allowing you to write how you want to write</p>
+					</div>
+				</div>
+				<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Handwritten.png">
```

Comment:
Writing it would take way too long. A 15 second GIF of the 1600px size is already 2 MB in size. It is possible to speed the video up to make the GIF short, but then people would barely know what they are seeing.

Reply:
Do we really require the 1600px? You can downsize by factor 1.5 which should get you below 1MB most likely..

Reply:
We could downscale, but then it would look subpar on higher resolution monitor, where the images are scaled to a maximum width of 1600px. @Technius already said that on higher resolution monitor, there's already plenty of whitespace, so reducing that wouldn't be an option.

Reply:
Hmm if we would have another system we could lazy-load the GIF. But that seems to be more hassle than it is worth. Let's leave it like it is.